.github/workflows/releasecandidate.yml

```diff
@@ -67,3 +67,17 @@ jobs:
           slack_webhook_url: ${{secrets.SLACK_WEBHOOK_URL}}
           channel: '#team-devops'
           name: 'Workflow Status'
+    
+  run-sdk-networking-tests:
+    needs: [test, staticbuild, appimage]
+    if: ${{ success() }}
+    name: Trigger Networking Tests Workflow
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Workflow
+        run: |
+          curl -X POST \
+          -H "Authorization: token ${{secrets.GITHUB_TOKEN}}" \
+          -H "Accept: application/vnd.github.everest-preview+json" \
```

Comment:
per [this](https://developer.github.com/changes/3/) I think everest-preview is no longer current

from [here](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-dispatch-event) I think it's okay to use vnd.github+json now